docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+    :alt: Update permission JSON
+    :target: `connect-azure-6.png`_
+    :align: center
+    :figclass: align-center
+
+Step 1.3: Assigning the custom role to Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Back in the *Subscription* or *Resource Group* overview, select *Add* and choose *Add role assignment*:
+
+.. _connect-azure-7.png: ../../../_images/connect-azure-7.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-7.png
+    :alt: Add role assignment
+    :target: `connect-azure-7.png`_
+    :align: center
+    :figclass: align-center
+
+Choose the custom role you just created, select *User, group, or service principal* to *Assign access to*
+and select the *hopsworks.ai* service principal. Press *Save*:
+
+.. _connect-azure-8.png: ../../../_images/connect-azure-8.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-8.png
+    :alt: Configure Hopsworks.ai as role assignment
+    :target: `connect-azure-8.png`_
+    :align: center
+    :figclass: align-center
+
+Go back to the Hopsworks.ai Azure account configuration workflow and proceed to the next step. Copy the CLI command shown:
+
+.. _connect-azure-9.png: ../../../_images/connect-azure-9.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-9.png
+    :alt: Configure subscription and tenant id
+    :target: `connect-azure-9.png`_
+    :align: center
+    :figclass: align-center
+
+Paste the CLI command into your terminal and execute it. Note that you might have multiple entries listed here.
+If so, ensure that you pick the subscription that you want to use.
+
+.. _connect-azure-10.png: ../../../_images/connect-azure-10.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-10.png
+    :alt: Show subscription and tenant id
+    :target: `connect-azure-10.png`_
+    :align: center
+    :figclass: align-center
+
+Copy the value of *id* and paste it into the *Subscription id*
+field on Hopsworks.ai. Go back to the terminal and copy the value of *tenantId*. Ensure to NOT use the *tenantId* under *managedByTenants*.
+Paste the value into the *Tenant ID* field on Hopsworks.ai and press *Finish*.
+
+Congratulations, you have successfully connected you Azure account to Hopsworks.ai.
+
+.. _connect-azure-10.1.png: ../../../_images/connect-azure-10.1.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-10.1.png
+    :alt: Store subscription and tenant id
+    :target: `connect-azure-10.1.png`_
+    :align: center
+    :figclass: align-center
+
+Step 2: Deploying a Hopsworks cluster
+-------------------------------------
+
+In Hopsworks.ai, select *Create cluster*:
+
+.. _create-instance.png: ../../../_images/create-instance.png
+.. figure:: ../../../imgs/hopsworksai/create-instance.png
+    :alt: Create a Hopsworks cluster
+    :target: `create-instance.png`_
+    :align: center
+    :figclass: align-center
+
+Select the *Location*, *Resource Group*, *Instance type* and *Local storage* size, name the cluster and proceed:
```

Comment:
I think Jim prefers us to explicitly say "click on next" rather than proceed.

Reply:
done